platforms/jvm/toolchains-jvm-shared/src/main/java/org/gradle/jvm/toolchain/internal/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Implementations and related internal support types for JVM toolchains.
+ */
+package org.gradle.jvm.toolchain.internal;
```

Comment:
Seems you cannot annotate with `NonNullApi` a package that was already annotated previously, this is because the `toolchain-jvm` and `toolchain-jvm-shared` are sharing the same package. This due on purpose to avoid more code refactor after moving those classes under a separate module. 

```
/Users/vmadalin/Develop/gradle/platforms/jvm/toolchains-jvm-shared/src/main/java/org/gradle/jvm/toolchain/package-info.java:20: error: package org.gradle.jvm.toolchain has already been annotated
@NonNullApi
^
1 error
14 warnings

> Task :docs:javadocAll FAILED
```